doc/modules/kernel_approximation.rst

```diff
@@ -247,7 +247,7 @@ or store training examples.
       <https://www.robots.ox.ac.uk/~vgg/publications/2010/Sreekanth10/sreekanth10.pdf>`_
       Vempati, S. and Vedaldi, A. and Zisserman, A. and Jawahar, CV - 2010
     .. [PP2013] `"Fast and scalable polynomial kernels via explicit feature maps"
-      <:doi:`10.1145/2487575.2487591`>`_
+      <https://doi.org/10.1145/2487575.2487591>`_
```

Comment:
I think the point is to use the `:doi:` directive outside of a restructured text link. So the title itself would not be clickable anymore.

Reply:
Thanks. Please review again.